graphql/src/datasources/es.ts

```diff
@@ -8,13 +8,18 @@ const ES_ADMINISTRATIVE_DIVISION_INDEX = 'administrative_division';
 const ES_HELSINKI_COMMON_ADMINISTRATIVE_DIVISION_INDEX =
   'helsinki_common_administrative_division';
 const ES_ONTOLOGY_TREE_INDEX = 'ontology_tree';
+const ES_ONTOLOGY_WORDS_INDEX = 'ontology_words';
```

Comment:
Other index names are singular (for some reason), so it would be nicer if this was too.

Reply:
renamed